src/types.ts

```diff
@@ -96,7 +96,7 @@ export interface QuizReturnState {
 
 export type AnswerInput = {
   type: InputQuestionsTypes;
-  value: string | string[];
+  value: string | Partial<QuestionOption>[];
```

Comment:
Will this ever have any other props other than id and value? If not can we use Omit<> instead and omit the unused props?

Reply:
True 👍